tests/bootstrap

```diff
@@ -50,27 +50,16 @@ EOF
 
 source "${SCRIPT_DIR}/bootstrap_utils.sh"
 
-PARAMS=$(getopt -o "o:h" --name ${BASENAME} -- "$@")
-
-eval set -- "$PARAMS"
-unset PARAMS
-
-while true ; do
-    case $1 in
-        -o)
-            BUILDDIR=$2
-            shift 2
+while getopts "o:h" opt ; do
+    case ${opt} in
+        o)
+            BUILDDIR="${OPTARG}"
             ;;
-        -h)
+        h)
             usage
             exit
             ;;
-        --)
-            shift
-            break
-            ;;
         *)
-            echo "Unrecognized option $1"
             usage
             exit 1
             ;;
```

Comment:
After the case, please reset the script arguments so that non-option arguments are available. We don't currently have any, but we should reset anyway.

shift $((OPTIND-1))